### PR TITLE
Sensu puppet

### PR DESCRIPTION
**Pull request by LeonLee88**
The ruby module has been added to ensure the libraries regarding with sensu will be installed

### CODE REVIEW

File: quickstack/manifests/compute_common.pp [hunk 1]
```diff
@@ -284,4 +287,36 @@
            nova_uuid     => $nova_uuid,
            rbd_key       => $rbd_key,
         }
+#Custimization for ruby. Ruby is necessary dependency of sensu
```
Comment (ravisantoshgudimetla) [inline quickstack/manifests/compute_common.pp]: I think we already have ruby gems. Can you please tell why do we need updated ruby gems.

File: sensu/manifests/init.pp [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+#   String.  Sensu log level to be used
+#   Default: 'info'
+#   Valid values: debug, info, warn, error, fatal
+#
+# [*init_stop_max_wait*]
+#   Integer.  Number of seconds to wait for the init stop script to run
+#   Default: 10
+#
+class sensu (
+  $version                     = 'latest',
```
Comment (ravisantoshgudimetla) [inline sensu/manifests/init.pp:223]: Are we using the parameters here? If yes, we need to move them to hiera as they contain username and password parameters.

File: quickstack/manifests/compute_common.pp (hunk 1 above)
Comment (LeonLee88) [inline quickstack/manifests/compute_common.pp]: @ravisantoshgudimetla I failed to install one necessary gem for running sensu plugin before with error:

 "ERROR: Could not find a valid gem 'sensu-plugin' (>= 0), here is why: Unable to download data from https://rubygems.org/ ".

Then I tried updating the packages list and  it works afterward. So I think I have to make sure the rubygem would have the latest package list, that could be achieve by running "gem update --system".

File: sensu/manifests/init.pp (hunk 2 above)
Comment (LeonLee88) [inline sensu/manifests/init.pp:223]: yes we are. We had done that already.

**Pull request closed without merging**